### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jhogrefe**
I found it was not working for me until I wrapped the variables in parentheses, so I thought it might be helpful to update the ReadMe here for others who might be running into the same problem.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,9 +68,9 @@ translator = BingTranslator.new('YOUR_CLIENT_ID', 'YOUR_CLIENT_SECRET')
 spanish = translator.translate 'Hello. This will be translated!', :from => 'en', :to => 'es'
 
 # without :from for auto language detection
-spanish = translator.translate 'Hello. This will be translated!', :to => 'es'
+spanish = translator.translate('Hello. This will be translated!', :to => 'es')
 
-locale = translator.detect 'Hello. This will be translated!' # => :en
+locale = translator.detect('Hello. This will be translated!' # => :en)
```
Comment (tsion) [inline README.md]: Your `)` is in a comment here.

**Comment by relrod** [comment]
@jhogrefe can you change the one (`audio`) below it as well, and maybe rebase them into one commit? Also, just curious what environment are you using that requires parentheses?

**Comment by jhogrefe** [comment]
I updated the audio one as well! I apologize but I haven't learned how to
rebase in git yet; I hope that is ok...

The project I created was done using Ruby/Sinatra, and I just deployed an
early version online:

http://www.freesoftwaretranslation.com

This is a website of software terms I provided translations for in a
database, and anything not found in the database is then pushed through the
Bing translator gem/API. Would love to hear your thoughts on it if you have
time to quickly check it out!

Best,

Jason

**Comment by relrod** [comment]
What Ruby version? It's odd that you need parentheses for those examples. Anyway, I will merge this, thanks for the patch!

**Comment by jhogrefe** [comment]
I am running Ruby 2.2.0.

Not sure why it didn't work without parentheses but on a whim I wrapped it
up and it worked.

Happy to help!

Jason

**Comment by relrod** [comment]
(Closing -- this was cherry-picked into 6d374e9b4fd186c6ba26043c656d14be5fe9307a). Thanks again!

**Pull request closed without merging**